File: 816-design-hashset/solution.cpp
class MyHashSet {
private:
    vector<bool> data;
public:
    /** Initialize your data structure here. */
    MyHashSet() {
        data = vector<bool>(1000000, false);
    }
    
    void add(int key) {
        data[key] = true;
    }
    
    void remove(int key) {
        data[key] = false;
    }
    
    /** Returns true if this set contains the specified element */
    bool contains(int key) {
        return data[key];
    }
};

/**
 * Your MyHashSet object will be instantiated and called as such:
 * MyHashSet obj = new MyHashSet();
 * obj.add(key);
 * obj.remove(key);
 * bool param_3 = obj.contains(key);
 */

File: 816-design-hashset/solution.py
class MyHashSet(object):

    def __init__(self):
        """
        Initialize your data structure here.
        """
        self._data = [False] * 1000000

    def add(self, key):
        """
        :type key: int
        :rtype: void
        """
        self._data[key] = True

    def remove(self, key):
        """
        :type key: int
        :rtype: void
        """
        self._data[key] = False
        

    def contains(self, key):
        """
        Returns true if this set contains the specified element
        :type key: int
        :rtype: bool
        """
        return self._data[key]


# Your MyHashSet object will be instantiated and called as such:
# obj = MyHashSet()
# obj.add(key)
# obj.remove(key)
# param_3 = obj.contains(key)